vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -64,10 +71,74 @@ private void setComponentStyles() {
 
   public void projectId(String projectId) {
     handler.projectId(projectId);
-
   }
-  public void setStyles(String... componentStyles){
+
+  public void setStyles(String... componentStyles) {
     getContent().addClassNames(componentStyles);
   }
 
+  /**
+   * Component logic for the {@link ProjectDetailsComponent}
+   *
+   * @since 1.0.0
+   */
+  private final class Handler {
+
+    private final ProjectDetailsComponent component;
+    private final ProjectInformationService projectInformationService;
+
+    private ProjectId selectedProject;
+
+    public Handler(ProjectDetailsComponent component,
+        ProjectInformationService projectInformationService) {
+      this.component = component;
+      this.projectInformationService = projectInformationService;
+    }
+
+    public void projectId(String projectId) {
+      projectInformationService.find(ProjectId.parse(projectId)).ifPresentOrElse(
+          this::loadProjectData,
+          () -> component.titleField.setValue("Not found"));
+    }
+
+    private void loadProjectData(Project project) {
+      this.selectedProject = project.getId();
+      component.titleField.setValue(project.getProjectIntent().projectTitle().title());
+      component.projectObjective.setValue(project.getProjectIntent().objective().value());
+      project.getProjectIntent().experimentalDesign().ifPresentOrElse(
+          experimentalDesignDescription -> component.experimentalDesignField.setValue(
+              experimentalDesignDescription.value()),
+          () -> component.experimentalDesignField.setPlaceholder("No description yet."));
```

Comment:
I think JB is still investigating on how to best use placeholders. I asked and we agreed that the best course of action would be to leave the field empty instead of showing a placeholder. 
N(IT: It's an experimental design not a description)
```suggestion
          () -> component.experimentalDesignField.setPlaceholder("No description yet."));
```

Reply:
I did not implement this and only moved it. Thus I will not change it in this PR